health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -481,6 +481,47 @@ const createUpdatePlanProject = async (req) => {
+        ?.map((row) => {
+          const updatedRow = { ...row };
+          const operatorName = row?.operatorName;
+          delete updatedRow?.operatorName;
+          updatedRow.operator = state?.RuleConfigureOperators?.find((operation) => operation.operatorName === operatorName)?.operatorCode;
+          return updatedRow;
+        });
+        //mix the current + api res
+        const upatedPlanObjSubFormula = {
+          ...fetchedPlanForSubFormula,
+          operations: [...prevFormulaValues, ...formulasToUpdateFromUIForSubFormula],
+        };
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)